sdk/appconfiguration/Azure.ApplicationModel.Configuration/tests/ConfigurationRecordedTestSanitizer.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Testing;
+
+namespace Azure.ApplicationModel.Configuration.Tests
+{
+    public class ConfigurationRecordedTestSanitizer : RecordedTestSanitizer
+    {
+        public override void SanitizeConnectionString(ConnectionString connectionString)
+        {
+            const string secretKey = "secret";
+
+            // Configuration client expects secret to be base64 encoded so we can't use the placeholder
+            if (connectionString.Pairs.ContainsKey(secretKey))
+            {
+                connectionString.Pairs[secretKey] = "";
```

Comment:
Can we add `[SANITIZED]` as the value to clue people in that we've modified it?

Reply:
The secret is expected to be base64 encoded or client ctor would throw, I could put base64 encoded `Sanitized` in there but I don't think it would help a lot.

Reply:
Fair enough.